docs/yamlpages/options.rst

```diff
@@ -132,7 +136,21 @@ value (``auto``) is equivalent to ``mixed`` when the device support this precisi
 
 Valid options: [auto]/double/mixed/single
 
-|
+.. _yaml_options_switch_experiment_interval:
+
+switch_experiment_interval
+--------------------------
+.. code-block:: yaml
+
+   options:
+     switch_experiments_interval: 0
+
+When running multiple experiments using the ``!Combinatorial`` tag, this allows to switch between experiments every
+``switch_experiments_interval`` iterations, and gather data about multiple molecules/conditions before
+completing the specified ``number_of_iterations``. If 0 or less, YANK will complete the combinatorial calculations
```

Comment:
I'm not a big fan of the "0 *or less*" being explicitly documented. I would say just "0" and leave the code to handle less in case someone messes up. Otherwise they have to think about "What does a negative interval mean?"